astropy/__init__.py

```diff
@@ -11,8 +11,11 @@
 import os
 from warnings import warn
 
-if sys.version_info[:2] < (3, 5):
-    warn("Astropy does not support Python <3.5 (in v3.0 and later)")
+__minimum_python_version__ = '3.5'
+__minimum_numpy_version__ = '1.9.0'
+
+if sys.version_info < tuple((int(val) for val in __minimum_python_version__.split('.'))):
```

Comment:
can't we use `LooseVersion` here? (from `distutils` or so).

Reply:
Hmm... Maybe?  I thought the recommended way with `sys.version_info` was to do the direct comparison, which is why `version_info` is such a weird tuple format... And you can't directly compare ``sys.version_info`` or ``sys.version`` with the distutils version objects.  But that predates `LooseVersion`, so maybe there's a way to get a version object that's the current python?

Reply:
You're right. `sys.version` shouldn't be used for extracting the `version`. And `LooseVersion('.'.join(map(str, sys.version_info)))` is also ugly...

Reply:
The whole version-comparison machinery is kind of a mess, sadly. I really *want* to use `LooseVersion` or similar for everything but there keeps being weird corner cases and changes to the rules that makes it hard to account for.